scripts/globals/abilities/pets/sleepga.lua

```diff
@@ -5,15 +5,19 @@ require("scripts/globals/monstertpmoves")
 require("scripts/globals/status")
 require("scripts/globals/magic")
 require("scripts/globals/msg")
+require("scripts/globals/spell_data")
```

Comment:
`spell_data` is already required by `magic`, no big deal, just reference for next time 👍